Vagrantfile

```diff
@@ -74,6 +76,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # consul_retry_join e.g. "172.31.1.11", "172.31.1.12", "172.31.1.13"
   consul_retry_join = master_infos.map{|master| "\"#{master[:ip]}\""}.join(", ")
 
+  # Ansible variables
+  ansible_extra_vars = {
+    zookeeper_peers: zookeeper_peers,
+    zookeeper_conf: zookeeper_conf,
+    consul_join: consul_join,
+    consul_retry_join: consul_retry_join,
+    mesos_master_quorum: conf['mesos_master_quorum'],
+    consul_bootstrap_expect: conf['consul_bootstrap_expect']
+  }
+  # Apollo environment variables
+  apollo_vars = get_apollo_variables(ENV)
+  # Add apollo variables to ansible ones
+  ansible_extra_vars = ansible_extra_vars.merge(apollo_vars)
```

Comment:
You could just do "ansible_extra_vars.merge!(apollo_vars)" for adding to ansible_extra_vars